.circleci/config.yml

```diff
@@ -103,5 +143,8 @@ workflows:
           filters:
             branches: { ignore: /.*/ }
             tags: { only: /.*/ }
-
-
+      - react_deploy_tag:
```

Comment:
So a couple things here:
1. This job will run without a valid workspace to attach to (you have to have the `react_build_storybook` job listed as a job in this workflow too.
2. It looks like this always releases to the root of the S3 bucket.  Just a heads up we sometimes do experimental tags (eg: 6.0.0-alpha1), which would cause the bucket to get updated with something that may or may not be stable.  That's probably fine for NPM (you may have a use case for a pre-release tag), but I'd imagine you want to be more intentional with what happens in the S3 bucket.

Re: S3 bucket, maybe this is good enough for the first iteration.   It would be really great if we could sync up on the S3 publishing between React and Patternlab and handle them the same way, which probably means a script that can detect whether a given tag is stable or not and sync to a subdirectory for non-stable tags.

Reply:
Thanks @rbayliss I fixed up the build_tag jobs list in ae4b37235

Also added an ignore value to react_deploy_tag that should ignore versions with /-.*/ after the number, I think?

Reply:
I think that regex needs an escape in it.  `-` is a special regex character.  Try `\-`

Reply:
Also, yeah, the addition of the job looks good to me.

Reply:
pedant moment: I think that's only true in a bracketed range (eg. [a-b]), but yea, escaping it to be 100% explicit doesn't hurt :) added in be3ebd9cf

Reply:
Ha, fair enough.  You're probably right - I get nervous with the tag workflows though, since we don't see them run until we're in the middle of a release.